ontology/yaml/resources/HVAC/entity_types/CU.yaml

```diff
@@ -43,6 +43,21 @@ CU_DFSS_DFVSC_CHWZTC_HWZTC_RTM_ZHM:
   - RTM
   - ZHM
 
+CU_DFSS_DFVSC_CHWZTC_HWZTC_RTM_ZHM_CO2CDSP_PDSCV_RDC:
+  description: "Compartment unit on discharge side with start/stop, status, cooling and heating (zone temp control), CO2 control with supply and return dampers."
+  is_canonical: true
+  implements:
+  - CU
+  - DFSS
+  - DFVSC
+  - CHWZTC
+  - HWZTC
+  - RTM
+  - ZHM
+  - CO2CDSP
+  - PDSCV
```

Comment:
Can you please clarify: is this damper upstream of the discharge fan?

Reply:
The supply damper is upstream indeed. The air from the DOAS go through the damper first then through the discharge fan.

Reply:
ok thank you for clarifying